app/presenters/nav_presenter.rb

```diff
@@ -63,4 +63,14 @@ def active_action?(subnav)
 
     subnav[:actions].include?(@params[:action])
   end
+
+  def first_team
+    teams.first
+  end
+
+  def teams
+    return [] unless @current_user
+
+    @current_user.memberships.where(role: 'lead').map(&:user_group)
```

Comment:
Doable in 1 query. Could be nice to obscure this in a `User#led_user_groups` method:

```suggestion
    @current_user.user_groups.merge(Membership.lead)
```